irohad/main/irohad.cpp

```diff
@@ -159,6 +172,62 @@ static shared_model::crypto::Keypair getKeypairFromFile(
              });
 }
 
+boost::optional<std::string> getEnvVar(std::string const& key)
```

Comment:
We highly recommend to use std:: alternatives instead of boost:: if they are.
```suggestion
std::optional<std::string> getEnvVar(std::string const& key)
```

Reply:
@kuvaldini Done: b86c880f49be3a638200af50c55010fc1198f3d6